js/core/set_template_engine.js

```diff
@@ -1 +1,2 @@
-module.exports = require('./templates/template_engine_registry').setTemplateEngine;
+import { setTemplateEngine } from './templates/template_engine_registry';
+export default setTemplateEngine;
```

Comment:
You can use re-export
```suggestion
export { setTemplateEngine as default } from './templates/template_engine_registry';
```